src/aac/spec/spec.yaml

```diff
@@ -280,6 +280,13 @@ data:
       type: Behavior[]
       description: |
         A list of behaviors that the system being modeled will perform.
+    - name: state
+      type: Field[]
+      description: |
```

Comment:
Will there be follow-up stories to validate (e.g.) that state is not accessed from other models, etc? If not, how do we prevent this (assuming this is something we *should* be trying to prevent)?

Reply:
I don't see this as something we validate in the model, but rather establish as modeling guidance.  Access to a model's state in practical application is achieved by not writing a code generator that exposes the state of a service externally.  If a modeler desires for some element of state to be exposed a behavior would have to be defined to accomplish that.  I'm thinking once we get our UI going we should start to focus on code generation to establish demonstratable examples of the modeling guidance such as this.